doc/Uplink.xml

```diff
@@ -359,10 +359,11 @@
           <para>Error : Optional user readable error information (readonly).</para>
         </listitem>
       </itemizedlist>
-      <para>The device shall interpret the field RemoteAddress as key to a specific uplink
-        configuration.</para>
-      <para>A device shall reject a configuration containing both CertificateID and
-        AuthorizationServer. </para>
+      <para>A device shall interpret the field RemoteAddress as key to a specific uplink
+        configuration and it shall support configuring the opional parameters CertificateID and
+        CertPathValidationPolicyID.</para>
+      <para>A device signaling AuthorizationModes via its capabilities shall support configuration
+        of the AutorizationServer.</para>
```

Comment:
Should this not state that if it signals 'jwt' in AuthorizationModes it must support configuring the AuthorizationServer?